### DISCUSSION TITLE
Autohide sidebars when scrolling down for distraction-free reading experience

### DISCUSSION THREAD

**Discussion by mfridman** [squidfunk/mkdocs-material#6359] [Ideas]
For a distraction-free reading experience, it'd be awesome to have an **autohide sidebar(s)** flag (particularly useful for blog posts).

There's already a really smooth [header.autohide](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/?h=autohide#automatic-hiding) feature. Could this be extended to the left/right sidebar content to "smoothly" be hidden as the user is scrolling down, but re-appear if scrolling up? 

Hopefully, this image helps clarify the intention:

![CleanShot 2023-11-18 at 08 56 34@2x](https://github.com/squidfunk/mkdocs-material/assets/6278244/ceb1e782-8469-440d-9cc7-2fa7bb904473)

**Comment by squidfunk** [comment]
Thanks for suggesting. Could be an extension to #3428, once we implement it. I've a working prototype, but I'm not happy with the mechanics yet. You can add it as a suggestion to #3428, if you like.

**Reply by mfridman** [reply]
Ugh, sorry. I really did search but GitHub failed me.

Let's close this in favour of the linked issue.
